From d573744953ed39e444510b35498e9d30fb874275 Mon Sep 17 00:00:00 2001
From: Dragan Marinkovic <dmarinkovi@sierrawireless.com>
Date: Thu, 23 Aug 2018 17:49:01 -0700
Subject: [PATCH] linux, kernel: Fix kernel crash if QCA9377 is enabled

0003-QCA9377-Support-ignore-resource-error.patch was introduced
to work around crash caused by QCA9377 support enablement, and
it should be removed.

Once QCA driver is enabled, it needs to take over subsys0.

Change-Id: I720ff7ead88ee3501ffd502348e68ffc46c1fa3b
Resolves: LXSWIREF-776
Signed-off-by: Dragan Marinkovic <dmarinkovi@sierrawireless.com>
---
 arch/arm/boot/dts/qcom/mdm9607-sierra-monitor.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/qcom/mdm9607-sierra-monitor.dtsi b/arch/arm/boot/dts/qcom/mdm9607-sierra-monitor.dtsi
index dfad51429730..397704156b8c 100644
--- a/arch/arm/boot/dts/qcom/mdm9607-sierra-monitor.dtsi
+++ b/arch/arm/boot/dts/qcom/mdm9607-sierra-monitor.dtsi
@@ -24,12 +24,12 @@
 
 		/* check if mpss is loaded */
 		sierra,monitor_resource@2 {
-			sierra,path = "/sys/bus/msm_subsys/devices/subsys0/state";
+			sierra,path = "/sys/bus/msm_subsys/devices/subsys1/state";
 			sierra,value = "ONLINE";
 			sierra,force-dload = <1>;
 		};
 		sierra,monitor_resource@3 {
-			sierra,path = "/sys/bus/msm_subsys/devices/subsys0/name";
+			sierra,path = "/sys/bus/msm_subsys/devices/subsys1/name";
 			sierra,value = "modem";
 			sierra,force-dload = <1>;
 		};
-- 
2.18.0

